tests/cases/compiler/jsxNamespaceImplicitImportJSXNamespace2.tsx

```diff
@@ -0,0 +1,46 @@
+// @strict: true
```

Comment:
this file should, of course, be renamed - but until I know more about the nature of the fix I'm not sure what it should be